stories/navigation-panel.stories.js

```diff
@@ -0,0 +1,1320 @@
+// https://github.com/EnCiv/undebate-ssp/issues/16
+
+import React, { useEffect } from 'react'
+
+import NavigationPanel from '../app/components/navigation-panel'
+
+export default {
+    title: 'Navigation Panel',
+    component: NavigationPanel,
+    argTypes: {},
+}
+
+const Template = (args, context) => {
+    const { onDone, electionOM } = context
+    const { defaultElectionObj, ...otherArgs } = args
+    const [electionObj, electionMethods] = electionOM
+    useEffect(() => electionMethods.upsert(defaultElectionObj), [defaultElectionObj])
+
+    return (
+        <div>
+            <NavigationPanel electionOM={electionOM} onDone={onDone} {...otherArgs} />
+        </div>
+    )
+}
+
+export const ElectionDefault = Template.bind({})
+ElectionDefault.args = {
+    defaultElectionObj: {
+        _id: '2349099238402',
+    },
+}
+
+export const ElectionComplete = Template.bind({})
+ElectionComplete.args = {
+    defaultElectionObj: {
+        _id: '2349099238402',
+        electionName: 'U.S Presidential Election',
+        organizationName: 'United States Federal Government',
+    },
+}
+
```

Comment:
I have a suggestion to shorten this file.  create 
```
const defaultElectionObj={ ...} in the file.
 ElectionComplete.args={ defaultElectionObj: {...defaultElectionObj, {timeline: {a different timeline}}}
```
this way you won't have to repeat stuff that doesn't change, and you focus attention on what's different.
If you wanted to go further down this path you could use
````
import { merge } from 'lodash'
ElectionComplete.args={
    defaultElectionObj: merge({},defaultElectionObj, 
          {timeline: { moderatorDeadlineReminderEmails:  { date: 'a different date' } }}
     )
}
```
This is not a required change.  But I wanted to mention it for future thinking.  
Thanks for your hard work!